### PR TITLE
Fixes #705 : Resume Builder WebApp

### LINKED ISSUE #705: [Feature Request]: Resume Builder WebApp Project (CRUD)
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

Nowadays, everyone, ranging from a student to a working professional, needs a well-built resume to apply to companies for jobs and internships. A resume not only reflects their skills but also reflects their personality and character. Companies believe much on first impression, and a good resume is what impresses them. But, the applicants either become unsuccesful to create a good resume or it takes a lot of time finding a good template for resume, and edit it.

### Use Case

- The Resume Builder would make the resume building process a cakewalk!
- Just enter the details like, name, personal details, academic details, work experience, skills, etc and boom, the resume will be ready in just a click!
- Also, the user will be able to update the details dynamically, including adding or removing sections, update personal information, or update new skills that they've learnt.
- More interestingly, the users will also be able to download their generated resume as PDF.

### Benefits

The Resume Builder extension will allow users to create their own resume in a hassle-free way, download the resume as pdf and also update their skills or personal details later, as per requirements. The feature will be completed when users will be able to enter their details, add or remove sections, update the details dynamically, and also generate and download a pdf of their resume.

### Add ScreenShots

_No response_

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by Sambit-Mondal**
## Related Issue

Fixes the issue no. #705 

## Description

The Resume Builder WebApp consists of the following features:
- It would make the resume building process a cakewalk!
- Just enter the details like, name, personal details, academic details, work experience, skills, etc and boom, the resume will be ready in just a click!
- Also, the user will be able to update the details dynamically, including adding or removing sections, update personal information, or update new skills that they've learnt.
- The users will also be able to view the live resume, while creating it.
- More interestingly, the users will also be able to download their generated resume as PDF.

## Type of PR

- [ ] Bug fix
- [ ] Feature enhancement
- [ ] Documentation update
- [x] Other (specify): Addition of a new project

## Screenshots / Videos (if applicable)

https://github.com/CodeHarborHub/codeharborhub.github.io/assets/138088550/c141e353-5535-45a4-a81b-b2a3457f6774

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

File: src/data/users.tsx [hunk 1]
```diff
@@ -292,14 +292,23 @@ const Users: User[] = [
     source: "https://github.com/ParnaRoyChowdhury777/Guess_The_Number_Game",
     tags: ["opensource", "product", "html", "css", "javascript", "vanilla"],
   },
+  {
+    title: "Resume Builder App",
+    description:
```
Comment (Ajay-Dhangar) [inline src/data/users.tsx:297]: Write your **description** limit only 120 characters

so, re-write

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update your code based on suggestions

**Comment by Sambit-Mondal** [comment]
@Ajay-Dhangar Done Sir. Kindly check.

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
@Sambit-Mondal you can close this PRs because I updated code for showcase projects data so, you need first sync or pull our latest main branch for new code then work on it.

**Pull request closed without merging**